recipes/watermark/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "watermark" %}
+{% set version = "1.3.1" %}
+{% set md5 = "cf25757d942e3a2ae5077b2ffff172dd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win and py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - ipython
+    - certifi
```

Comment:
Please drop `ipython` and `certifi` from build as they should not be needed. Unless I'm missing something.